src/media/css/elements--nav.styl

```diff
@@ -85,6 +85,15 @@ main:after {
     }
 }
 
+// We hide the recommendations tab by default and enable it via JS if user is
+// logged in and opted in to recommendations.
+[data-mkt-nav--item~="recommended"] {
```

Comment:
This should maybe belong outside of elements--nav.styl since it's Fireplace-specific.